### DISCUSSION TITLE
Biaxial compression test-Mohr coulomb fails

### DISCUSSION THREAD

**Discussion by ranjeet270** [idaholab/moose#27664] [Q&A Modules: Solid mechanics]
Hi,
I am trying to this Biaxial compression test from the PLAXIS module.  
<img width="261" alt="image" src="https://github.com/idaholab/moose/assets/64901303/fd64d029-a19d-4f3e-a365-10b1d49fe283"> <img width="421" alt="image" src="https://github.com/idaholab/moose/assets/64901303/bf052c5e-79a2-4715-b892-bfe92e4a2434">
<img width="359" alt="image" src="https://github.com/idaholab/moose/assets/64901303/820a3fc4-ac78-4c17-b3ba-184e015e0ad6">

[Here is my input file](https://github.com/ranjeet270/mc_test/blob/main/biaxial_compression.i) and I am getting error: 
Time Step 1, time = 0.1, dt = 0.1
 0 Nonlinear |R| = 8.495793e+00
      0 Linear |R| = 8.495793e+00
After reducing the stepsize to 0.0078125 with original strain increment with L2norm 2.36573e-09 the returnMap algorithm failed
++++++++++++++++++++++++
Checking the derivatives
++++++++++++++++++++++++
Debug Parameters are as follows
stress =
   -2.33717e-08     1.89488e-10               0
    1.89488e-10    -1.37257e-08               0
              0               0    -9.27434e-09

<img width="893" alt="image" src="https://github.com/idaholab/moose/assets/64901303/2273ec6e-1978-4135-8252-88bfb822b671">

**Comment by jmeier** [comment]
Dear @ranjeet270, please find this bi-axial test as moose-input file here:
https://github.com/jmeier/geotechnical-simulations-with-moose/tree/main/src/examples/bi-axial-shear
